apps/storybook/src/ComboBox.test.ts

```diff
@@ -13,6 +13,7 @@ describe('ComboBox', () => {
     'With Message',
     'With Status',
     'Multiple Select',
+    'Disabled',
```

Comment:
i'm surprised (and glad) this works now. initially we excluded it because it was taking too long to load (it was freezing up and timing out). i believe that is also the reason we excluded `Virtualized` (fixed in #986).

Reply:
Ohh, I see. To be sure, I ran this test a couple of times and it passed alright. So including this test.